docs/developer/community/faq.mdx

```diff
@@ -123,3 +123,29 @@ To install the latest stable versions of Saleor, storefront, and dashboard, plea
 ## How can I contribute to the project?
 
 Saleor is an open-source e-commerce platform. We provide support and guidance for our community. You can contribute by creating pull requests on [GitHub](https://github.com/mirumee/saleor/pulls). We always welcome your ideas. If you have any questions or feedback, you can also reach us on [Spectrum](https://spectrum.chat/saleor).
+
+## Something went wrong with the payment, how can I check the gateway response?
+
+The easiest way is to use API for fetching checkout payments with transactions and check such fields as `error` and `gatewayResponse`.
```

Comment:
Maybe we should add information on which permissions are required for this operation?